pkg/proto/dispatch/v1/01_codec.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2021 The Vitess Authors.
```

Comment:
This codec tells gRPC that it should try to use vtproto first and fallback to regular proto parsing.

Reply:
Can we add that as a comment somewhere in here?